Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.8-alpine3.6
+
+ENV CGO_ENABLED=0 \
+    PATH="$PATH:/app/node_modules/.bin/"
+
+RUN apk add --no-cache bash make zip chromium yarn && \
+    yarn install && \
+    mkdir -p $GOPATH/src/github.com/dannyvankooten && \
+    ln -s /app $GOPATH/src/github.com/dannyvankooten/browserpass
+
+WORKDIR /app
```

Comment:
could we name it `browserpass` instead of `app`? I might have hardcoded the folder name in the makefile :smile: